roles/pulp-webserver/templates/nginx.conf.j2

```diff
@@ -84,8 +84,13 @@ http {
         include conf.d/*;
 
         location / {
-            # checks for static file, if not found proxy to app
-            try_files $uri =404;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+            proxy_set_header X-Forwarded-Proto $scheme;
+            proxy_set_header Host $http_host;
+            # we don't want nginx trying to do something clever with
+            # redirects, we set the Host: header above already.
+            proxy_redirect off;
+            proxy_pass http://pulp-api;
```

Comment:
I expected this to use the `http://{{ pulp_api_host }}:{{ pulp_api_port }}` like the others do. I guess they could all be de-duplicated or used explicitly throughout?

Reply:
Actually dedplicating the others to be like this one is my recommendation since we do define it in one place above.